astropy/cosmology/tests/test_parameters.py

```diff
@@ -3,9 +3,10 @@
 # STDLIB
 from types import MappingProxyType
 
+import pytest
```

Comment:
This is a step too far? LoL. Isn't `pytest` technically third-party like `numpy`?

Reply:
Lol. yes. what's happening here is numpy has it's own section (I'm not sure why astropy had this historically) in the isort config, so pytest is getting moved up, but the section header is not. I'll just tweak that. 

I'd also rather not separate out numpy in the config...

Reply:
Pushing a new commit consolidating import headings

Reply:
Let's just keep the basic sorting part of isort and not try to make it too fancy? 😬

Reply:
Done. We can always re-implement the auto-labelling.